main.py

```diff
@@ -24,6 +24,9 @@ def trans_coverage_file(file, ext=None):
         with open(file, "r") as myfile:
             s = myfile.read()
 
+            if type(s) == bytes:
+                s = s.decode('utf8')
```

Comment:
perhaps, just add try: catch (do nothing) to make it safe? I have experienced this
exception with open.